tests/screenshot/apps/components/Button.js

```diff
@@ -256,6 +258,32 @@ const ButtonTests = [
 		<Button icon="arrowhookright" iconFlip="vertical">click me</Button>,
 		<Button icon="arrowhookright" iconFlip="both">click me</Button>,
 		<Button icon="arrowhookright" iconFlip="auto">click me</Button>
+	]),
+
+	// *************************************************************
+	// With customized button
+	// *************************************************************
+	// Note: When the file name of the test is too long, different tests may be recognized as the same test.
+	// So we changed the Button name of Focus prop tests.
+	...withProps({css: css}, [
+		// standard button
+		<Button>Customized button</Button>,
+		<Button selected>Customized button</Button>,
+		<Button size="small">Customized button</Button>,
+
+		// With icon.
+		<Button icon="minus" />,
+		<Button icon="minus" size="samll"> </Button>,
+		<Button icon="minus" selected> </Button>,
+		<Button icon="minus" iconPosition="after" />,
+		<Button icon="minus" iconPosition="after" size="small" />,
+
+		// With long text
+		<Button>Focused Lorem ipsum dolor sit amet, consectetur adipiscing elit.</Button>,
+		<Button selected>Focused Lorem ipsum dolor sit amet, consectetur adipiscing elit.</Button>,
+		<Button size="small">Focused Lorem ipsum dolor sit amet, consectetur adipiscing elit.</Button>,
```

Comment:
Do we need these TC for long text?

Reply:
I agree with you. I removed TC for long text.